src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -289,17 +476,79 @@
   word-break: break-word;
 }
 
-.pf-c-notification-drawer__group-toggle-count {
-  margin-right: var(--pf-c-notification-drawer__group-toggle-count--MarginRight);
-  margin-left: auto;
+// Task group toggle
+.pf-c-notification-drawer__task-group-toggle {
+  display: grid;
+  align-items: center;
```

Comment:
This causes the toggle and count to be centered vertically:

<img width="865" alt="Screen Shot 2021-08-04 at 5 34 56 PM" src="https://user-images.githubusercontent.com/35148959/128264129-b301f57b-a408-4a09-9a95-1762f9a763a2.png">

That doesn't match the existing notification group styling:

<img width="837" alt="Screen Shot 2021-08-04 at 5 33 38 PM" src="https://user-images.githubusercontent.com/35148959/128264159-7e1d07a8-1b1a-4662-847d-6fb5b3e815f5.png">

Reply:
The title in the task group toggle and the existing group toggle for alerts is actually clamped at 1 line as far as I can tell?

The center alignment  provided the alignment shown in the [mockup](https://marvelapp.com/prototype/g8idd54/screen/79182156) when there is a timestamp below the title, and the count and toggle icon are shown as center aligned vertically. I asked @mcarrano and if the title does actually go to multiple lines, we could top align instead.